lib/cog/util/command_helpers.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Cog.Command.Helpers do
+
+  def add_errors(input, error_or_errors),
```

Comment:
If you're going to do this, this module needs documentation about what's going on, what the assumptions are, etc. As is, it's not clear.